sig-docs/charter.md

```diff
@@ -0,0 +1,76 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs is responsible for Kubernetes documentation.
+
+The responsibility for feature documentation lies with feature developers. SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and coordinates documentation updates during quarterly releases.
+
+We maintain the Kubernetes website's infrastructure and tooling.
+
+We create subprojects as needed to handle specific aspects of Kubernetes documentation.
+
+### In scope
+
+SIG [readme]
+
+#### Code, Binaries and Services
+
+- The kubernetes.io website:
+
+  - Infrastructure and tooling
+  - Internationalization and localization support
+  - Documentation and release note training for other sigs
```

Comment:
Can you clarify this: "Documentation and release note training for other sigs" as "Documentation training and release note training for other sigs"

Reply:
Added.